_inc/client/at-a-glance/stats.jsx

```diff
@@ -59,8 +59,47 @@ const DashStats = React.createClass( {
 		return ( getSiteConnectionStatus( this.props ) === 'dev' ) ? demoStatsData : s;
 	},
 
+	/**
+	 * Checks that the stats fetching didn't return errors.
+	 *
+	 * @returns {object|bool}
+	 */
+	statsErrors() {
+		let checkStats = window.Initial_State.statsData.general;
+		if ( 'object' === typeof checkStats.errors ) {
```

Comment:
Oh I see, the error comes from the API response within stats_get_from_restapi().